app/javascript/packs/application.js

```diff
@@ -42,7 +42,7 @@ function handlers() {
     }
   })
 
-  document.querySelectorAll('.slide_open_href').forEach((link) => {
+  document.querySelectorAll('.slide_open_next').forEach((link) => {
```

Comment:
I believe profile editing currently uses this code with `.slide_open_href` and would break.